css/templates/land-see/page.css

```diff
@@ -351,6 +350,119 @@ amp-img.cover img {
   z-index: 10;
 }
 
-.ampstart-input input[type="email"].user-invalid {
+.land-see-subscribe-form input[type="email"].user-invalid {
   border: 2px solid var(--color-form-invalid);
 }
+
+/* Accordion */
+
+.land-see-story-category section {
+  margin-bottom: 10px;
+  position: relative;
+  overflow: hidden;
+}
+
+.land-see-story-category section > h3 {
+  text-transform: uppercase;
+  font-weight: bold;
+  min-height: 68px;
+  line-height: 68px;
+  outline: none;
+}
+
+.land-see-story-category section > h3:after {
+  content: '';
+  position: absolute;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='14' height='14' viewBox='0 0 14 14'%3E%3Cpath fill='none' stroke='%23021425' stroke-width='2' stroke-miterlimit='10' d='M6.967 13.932V0M0 6.966h13.932'/%3E%3C/svg%3E");
+  height: 14px;
+  width: 14px;
+  right: 30px;
+  top: 28px;
+  cursor: pointer;
+}
+
+.land-see-story-category section[expanded] > h3:after {
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='14' height='14' viewBox='0 0 14 14'%3E%3Cpath fill='none' stroke='%23021425' stroke-width='2' stroke-miterlimit='10' d='M0 6.966h13.932'/%3E%3C/svg%3E");
+}
+
+.land-see-story-category section:before {
+  content: '';
+  position: absolute;
+  height: 168px;
+  left:0;
+  right: 0;
+  background-color: var(--color-story-category-background);
+}
+
+.land-see .i-amphtml-accordion-header { /* reset defaults */
+  cursor: default;
+  background-color: inherit;
+  padding-right: 0;
+  border: none;
+}
+
+.land-see-story-category section.interiors:before {
```

Comment:
land-see-interiors

I guess you could come up with an additional class name on the category 

`land-see-story-interior` now you just need to use only one class to target

Sometimes adding extra classes would save you size

Reply:
Good suggestion, that'll help trim some CSS fat.